packages/asset-swapper/src/utils/market_operation_utils/constants.ts

```diff
@@ -1343,6 +1423,87 @@ export const NERVE_BSC_INFOS: { [name: string]: CurveInfo } = {
     },
 };
 
+export const SYNAPSE_BSC_INFOS: { [name: string]: CurveInfo } = {
+    [SYNAPSE_BSC_POOLS.nUSDLP]: {
+        exchangeFunctionSelector: CurveFunctionSelectors.swap,
+        sellQuoteFunctionSelector: CurveFunctionSelectors.calculateSwap,
+        buyQuoteFunctionSelector: CurveFunctionSelectors.None,
+        poolAddress: SYNAPSE_BSC_POOLS.nUSDLP,
+        tokens: [BSC_TOKENS.nUSD, BSC_TOKENS.BUSD, BSC_TOKENS.USDT, BSC_TOKENS.USDC],
```

Comment:
I'm not sure whether the order of the tokens matters here (it matters for curve pools afair). The order for this pool is nUSD, BUSD, USDC, USDT on Synapse website.